testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/web/security/authentication/BasicAuthenticationMechanismPicketboxRemovedTestCase.java

```
@@ -0,0 +1,188 @@
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.junit.Assert.fail;
+
+/**
+ * Automated test for [ WFLY-10282 ]
+ * Test Enables elytron and removes security subsystem. After secured EJB is deployed and accessed using basic authorization
+ * test checks if no NullPointerException is thrown.
+ *
+ * @author Daniel Cihak
+ */
+@RunWith(Arquillian.class)
+public class BasicAuthenticationMechanismPicketboxRemovedTestCase {
+
+    private static Logger LOGGER = Logger.getLogger(BasicAuthenticationMechanismPicketboxRemovedTestCase.class);
+
+    private static final String EJB_SECURITY = "ejb-security";
+
+    private volatile ModelControllerClient modelControllerClient;
+    private volatile CommandContext commandCtx;
+    private volatile ByteArrayOutputStream consoleOut = new ByteArrayOutputStream();
+
+    @Deployment(name = EJB_SECURITY)
+    public static WebArchive appDeployment1() {
+        WebArchive war = ShrinkWrap.create(WebArchive.class, EJB_SECURITY + ".war");
+        war.addClasses(BasicAuthenticationMechanismPicketboxRemovedTestCase.class, SecuredEJB.class, SecuredEJBServlet.class);
+        war.setWebXML(Utils.getResource("org/jboss/as/test/integration/web/security/authentication/deployment/web.xml"));
+        war.addAsWebInfResource(Utils.getResource("org/jboss/as/test/integration/web/security/authentication/deployment/jboss-web.xml"), "jboss-web.xml");
+        war.addAsWebInfResource(Utils.getResource("org/jboss/as/test/integration/web/security/authentication/deployment/beans.xml"), "beans.xml");
+        war.addAsResource(BasicAuthenticationMechanismPicketboxRemovedTestCase.class.getPackage(), "jboss-ejb-client.properties", "jboss-ejb-client.properties");
+
+        return war;
+    }
+
+    @Before
+    public void before() {
+        try {
+            initServerConfiguration(EJB_SECURITY);
+        } catch (Throwable t) {
+            LOGGER.error("Server configuration initialization failed.", t.getMessage(), t);
```

Don't do this, just let the exception propagate. It will have the same effect gives much more info as the full stack trace is in the surefire report.